src/content/pages/program.md

```diff
@@ -1,9 +1,272 @@
 ---
 title: Program
+# layout: "../layouts/Page.astro"
 ---
 
-# Program
+<!-- import Button from "../components/Button.astro" -->
 
-Our call for proposals for talks will be coming soon!
+# PyCon AU call for proposals
 
-=======
+We’d love you to present at PyCon AU 2024!
+
+**Our call for proposals will open on June 1st, 2024.**
+
+We’re looking for talks on everything Python, open source, and tech-related. If it would be of interest to someone, anyone, in the Python community, it’s of interest to us!
+
+The PyCon AU call for proposals (CFP) is open from June 1, 2024 until the end of **Sunday 21 July, 2024**, anywhere on earth. You can submit to the main conference and specialist tracks through the same process.
+
+You can submit your proposal through our online portal, Pretalx. A large button linking to the portal in a calming, brand-appropriate green colour will appear on this page when the call for proposals opens.
+
+<!-- <Button type="chonk" block href="https://pretalx.com/pyconau-2024/cfp">
+  Submit to the PyCon AU CFP!
+</Button> -->
+
+**If you can't afford to attend PyCon AU or don't have an employer that will sponsor your travel, please submit anyway! PyCon AU will have a financial aid program to assist attendees who otherwise couldn't make it to the conference, and we will prioritise speakers in that process.**
+
+We welcome submissions from everybody, including from those:
+
+- Who have never given a conference talk before,
```

Comment:
There are several dot-pointed lists on the page, but this is the only one appended with commas -- there is no doubt this is a grammatical choice and all good, but worth noting that it isn't internally consistent with other dot-point lists on the page. This is potentially fine.